### ISSUE TITLE
Similiary checks are not working (input)

### ISSUE THREAD

**Issue by hugowangler** [rpa-tomorrow/substorm-nlp#124]
# Expected behavior
When I input an instruction which contains a misspelled verb such as "schedle" the NLP module should compare the input word with the module verbs (I think?) and call the most similar module.

# Actual behavior
```
schedle a meeting with Mark at 15:00
./lib/nlp/nlp.py:37: UserWarning: [W008] Evaluating Doc.similarity based on empty vectors.
  similarity = doc_action.similarity(doc_verb)
I did not understand
```
# Steps to reproduce
Might not be reliable, I only get the warning the on first input after starting the CLI, but it never understands the input.
1.  start cli
2. input `schedle a meeting at 15:00`

# Additional context
When I look at the code in `nlp.py` (around line 37) there seems to be code which is supposed to handle this which is why I marked it as a bug.

**Comment by markhakansson** [comment]
I suggest moving the fuzzy search from automation to the NLP class, such that we do both similarity checks and misspelling checks.

**Comment by hugowangler** [comment]
Yeah this sounds better since it's some sort of data preprocessing it should be done before the NLP tries to interpret it.
